_functions_plf.sh

```diff
@@ -186,7 +186,13 @@ do_install_addons() {
     # Let's install them from $DEPLOYMENT_ADDONS env var
     _addons=$(echo $DEPLOYMENT_ADDONS | tr "," "\n")
     for _addon in $_addons; do
-      ${_addons_manager_script} install ${_addons_manager_option_catalog:-} ${_addon} ${_addons_manager_option_conflict:-} --force --batch-mode
+    # Install Onlyoffice addon using the --no-compat flag, may be resolved if catalog get updated.
```

Comment:
👍 the catalog must be fixed to avoid that

Reply:
Removed form code.
in catalog we currently have : 
{
    "id": "exo-onlyoffice",
    "version": "1.2.0",
    "unstable": false,
    "name": "Onlyoffice add-on",
    "description": "Edit office documents directly in eXo Platform",
    "releaseDate": "2018-06-10T22:00:00.000Z",
    "sourceUrl": "https://github.com/exo-addons/onlyoffice",
    "screenshotUrl": "https://raw.githubusercontent.com/exo-addons/onlyoffice/master/docs/images/action_bar.png",
    "documentationUrl": "https://github.com/exo-addons/onlyoffice/blob/master/README.md",
    "downloadUrl": "http://addons.exoplatform.org/exo/onlyoffice/exo-onlyoffice-editor-packaging/1.2.0/zip",
    "vendor": "eXo",
    "author": "Peter Nedonosko",
    "authorEmail": "pnedonosko@exoplatform.com",
    "license": "LGPLv3",
    "licenseUrl": "https://www.gnu.org/licenses/lgpl-3.0.txt",
    "mustAcceptLicense": false,
    "supportedDistributions": "community,enterprise",
    "supportedApplicationServers": "tomcat,jboss",
    "compatibility": "[5.0,5.1)"
  }
but AM still rise an error : 
[ERROR] The add-on exo-onlyoffice:1.2.0 is not compatible : Only eXo platform versions [5.0,5.1) are supported. Use --no-compat to bypass this compatibility check and install anyway

Reply:
resolved in catalog